src/Components/ServiceScene/LogsJsonScene.tsx

```diff
@@ -388,15 +434,50 @@ export class LogsJsonScene extends SceneObjectBase<LogsJsonSceneState> {
   };
 
   /**
-   * Gets re-root button and key label for root node when line format filter is active
+   * Gets re-root button and key label for root node when line format filter is active.
+   * aka breadcrumbs
    */
   private renderNestedNodeButtons = (keyPath: KeyPath, jsonFiltersSupported?: boolean) => {
+    const lineFormatVar = getLineFormatVariable(this);
+    const filters = lineFormatVar.state.filters;
+    const rootKeyPath = [DataFrameLineName, 0, VizRootName];
+
     return (
       <>
-        <span className={jsonLabelWrapStyles}>
-          {jsonFiltersSupported && <ReRootJSONButton keyPath={keyPath} setNewRootNode={this.setNewRootNode} />}
-          <strong>{this.getKeyPathString(keyPath)}</strong>
+        <span className={drillUpWrapperStyle} key={VizRootName}>
+          <Button
+            size={'sm'}
+            onClick={() => jsonFiltersSupported && this.setNewRootNode(rootKeyPath)}
+            variant={'secondary'}
+            fill={'outline'}
+            disabled={!filters.length}
+            name={keyPath[0].toString()}
+          >
+            {this.getKeyPathString(keyPath, filters.length ? '' : ':')}
+          </Button>
+          {filters.length > 0 && <span className={breadCrumbDelimiter}>{'>'}</span>}
```

Comment:
I would try what we use in the actual breadcrumb components which is `<Icon name="angle-right" />`

Reply:
Good call, fixed in https://github.com/grafana/logs-drilldown/pull/1151/commits/78368ba2cd2ad450dbdbeb9a078e225073f47817